### PR TITLE
Test value transfer states

### PR DESCRIPTION
**Pull request by zancas**
We should revisit including an "Expose/Deshield" kind for sending to transparent.

### CODE REVIEW

File: zingolib/src/wallet/data.rs [hunk 1]
```diff
@@ -752,16 +752,20 @@ pub mod summaries {
...
+    /// non_exhaustive to permit expanding to include an
+    /// Deshield variant fo sending to transparent
+    #[non_exhaustive]
     #[derive(Clone, Copy, PartialEq, Eq, Debug)]
     pub enum ValueTransferKind {
-        /// TODO: Add Doc Comment Here!
+        /// The recipient is different than this creator
         Sent,
-        /// TODO: Add Doc Comment Here!
+        /// The recipient is the creator and a shielded address
```
Comment (Oscar-Pepper) [inline zingolib/src/wallet/data.rs:763]: And the note includes a text memo

**Review by Oscar-Pepper — COMMENTED** [review]
Thanks for adding doc comments I noticed NoteToSelf was not fully defined

File: zingolib/src/wallet/data.rs (hunk 1 above)
Comment (zancas) [reply zingolib/src/wallet/data.rs:763]: Fixed.

**Comment by zancas** [comment]
Hey @fluidvanadium I think this is ready for review again.

**Review by Oscar-Pepper — APPROVED** [review]

**Comment by fluidvanadium** [comment]
HEY! Nobody even bothered to read my doc comments on this??

**Pull request merged**